content/maxwell3_fdem/MT/MT_N_layered_Earth.rst

```diff
@@ -0,0 +1,329 @@
+.. _MT_N_layered_Earth:
+
+Response of a layered Earth to a plane wave
+===========================================
+
+.. topic:: Purpose
+
+    We present here a 1D modelisation of the MagnetoTelluric waves in a layered Earth and the associated geophysical data. Our goal is to build better representation of the different physical phenomenons and better understanding of the resulting data.
+
+
+.. raw:: html
+    :file: ./images/movieMT_time.html
+
+Introduction
+------------
+
+We present here a 1D modelisation of the MagnetoTelluric waves in a layered Earth and the associated geophysical data. Our goal is to build better representation of the different physical phenomenons and better understanding of the resulting data. This work follows the derivation presented in :cite:`ward1988` and is supported by interactive apps developed in a `binder`_.
+
+ .. image:: http://mybinder.org/badge.svg 
+    :target: http://mybinder.org/repo/ubcgif/em_examples/notebooks/geophysical_surveys/MT_N_Layered_Earth/MT_n_layered_earth_example.ipynb
+    :align: center
+
+.. _binder: http://mybinder.org/repo/ubcgif/em_examples/notebooks/geophysical_surveys/MT_N_Layered_Earth/MT_n_layered_earth_example.ipynb
+
+MagnetoTelluric is a widely used method, especially for imaging geothermy. Its ability to image deep structure, up to several kilometers depth, is unique in EM geophysics. It is a passive method that use waves generated mostly in the Earth's Atmosphere. High frequencies are mainly produced by lightning strikes all around the globe, travelling through the Earth's Ionosphere that acts as a waveguide. Low Frequencies waves are produced through the interaction of the Earth's Ionosphere with solar wind and Earth's magnetic field.
+
+In MagnetoTellurics problems, the key diagnosed physical property is :ref:`electrical conductivity<electrical_conductivity_index>` :math:`\sigma`, as we expect the contrasts of the others physical properties to be negligible. 
+
+
+
+Setup
+-----
+
+ .. figure:: images/MT_N_layered_Earth-1.hires.png
+    :align: center
+    :scale: 50% 
+    :name: MTlayeredEarth
+
+    1D general Earth Model Configuration
+
+An example of a 1D MagnetoTelluric modelisation, with a 2-layers Earth, is shown in the movie above, where we have:
+
+ - a layered Earth, each layer with its own physical properties :math:`\sigma, \varepsilon, \mu`
+
+ - a plane wave traveling along the axis Z coming from air, composed of an electric field :math:`\mathbf{E_x}` and a magnetic field :math:`\mathbf{H_y}`. These are the fields we will be measuring with our geophysical instruments at the surface to obtain information from the underground
```

Comment:
Z is used later on as impedance, so I would suggest using ``:math:`\mathbf{\hat{z}}` `` here